images/test-environments/linux/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG CONDA_VER=latest
```

Comment:
A readme or similar in this directory could help explain what each of these three new Dockerfiles is used for.